pkg/controllers/applicationfailover/common_test.go

```diff
@@ -226,6 +226,15 @@ func Test_parseJSONValue(t *testing.T) {
 			wantErr: assert.NoError,
 			want:    "2",
 		},
+		{
```

Comment:
It would be nice to add a comment describing what the test case does.

Reply:
We can get it from the test case `get a static value`.

Reply:
Update with `get a static value just as jsonPath write`